src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -776,9 +776,25 @@ The accounting information can be accessed only through a *JMX client* or the Pr
 [[_run_as_me]]
 == Run Computation with a user's system account
 
-When starting a ProActive Node, it can be configured to run tasks under the user's system account when
-link:ProActiveUserGuide.html#_run_computation_with_your_system_account[runAsMe] is set to true in
-the job's definition. The default is to use the password method.
+Configure a ProActive Node to execute tasks under a user's system account by ticking the
+link:ProActiveUserGuide.html#_run_computation_with_your_system_account[Run as me] box in the task configuration.
```

Comment:
I did not write the documentation that way so I can just assume: 
There is another part about run as me in the user documentation, which is explicitly talking about how to create user credentials. 
Are you trying to say that I should remove the link?